[2024-08-21 19:13:14,940] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: captors_dag.task_2 2024-08-21T19:05:47.227767+00:00 [queued]>
[2024-08-21 19:13:14,958] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: captors_dag.task_2 2024-08-21T19:05:47.227767+00:00 [queued]>
[2024-08-21 19:13:14,959] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2024-08-21 19:13:14,959] {taskinstance.py:1043} INFO - Starting attempt 2 of 3
[2024-08-21 19:13:14,959] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2024-08-21 19:13:14,970] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): task_2> on 2024-08-21T19:05:47.227767+00:00
[2024-08-21 19:13:14,974] {standard_task_runner.py:52} INFO - Started process 53 to run task
[2024-08-21 19:13:14,978] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'captors_dag', 'task_2', '2024-08-21T19:05:47.227767+00:00', '--job-id', '7', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/mod_captors.py', '--cfg-path', '/tmp/tmpg_a0kcvj', '--error-file', '/tmp/tmpml6mhy85']
[2024-08-21 19:13:14,979] {standard_task_runner.py:77} INFO - Job 7: Subtask task_2
[2024-08-21 19:13:15,025] {logging_mixin.py:104} INFO - Running <TaskInstance: captors_dag.task_2 2024-08-21T19:05:47.227767+00:00 [running]> on host ad333fe07a6a
[2024-08-21 19:13:15,069] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=captors_dag
AIRFLOW_CTX_TASK_ID=task_2
AIRFLOW_CTX_EXECUTION_DATE=2024-08-21T19:05:47.227767+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-08-21T19:05:47.227767+00:00
[2024-08-21 19:13:15,079] {python.py:118} INFO - Done. Returned value was: None
[2024-08-21 19:13:15,091] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=captors_dag, task_id=task_2, execution_date=20240821T190547, start_date=20240821T191314, end_date=20240821T191315
[2024-08-21 19:13:15,122] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-21 19:13:15,152] {local_task_job.py:146} INFO - Task exited with return code 0
